pkg/bot/discord.go

```diff
@@ -0,0 +1,133 @@
+package bot
```

Comment:
Please add License header. You can copy it from other .go files just change the year to 2020

Reply:
resolved